models/quality_measures/quality_measures_models.yml

```diff
@@ -131,6 +131,57 @@ models:
       - name: tuva_last_run
         description: The date and timestamp of the dbt run.
 
+## Shared Exclusions
+  - name: shared_exclusions__frailty
```

Comment:
We need to rename all of the shared exclusion models so they follow the naming pattern for the rest of the data mart. Example: `quality_measures__shared_exclusions_frailty`, `quality_measures__shared_exclusions_advanced_illness`, etc.

Reply:
All of the shared exclusion models have been renamed to have `quality_measures__shared_exclusions` prefix